File: android/src/main/kotlin/uz/shs/video_player/models/Season.kt
package uz.shs.video_player.models

import com.google.gson.annotations.SerializedName
import java.io.Serializable

data class Season(
    @SerializedName("title")
    val title: String,
    @SerializedName("movies")
    val movies: List<Movie>,
) : Serializable

File: lib/models/season.dart
import 'package:video_player/models/movie.dart';

class Season {
  const Season({required this.title, required this.movies});

  final String title;
  final List<Movie> movies;

  Map<String, dynamic> toMap() => {'title': title, 'movies': movies.map((v) => v.toJson()).toList()};

  @override
  String toString() => 'Season{title: $title, movies: $movies}';
}
